tox.ini

```diff
@@ -5,7 +5,7 @@ envlist = py27,py34,py35,py36,py37,pypy,pypy3,docs,readme
 deps=
     -rrequirements-test.txt
 commands=
-    coverage run --source oauthlib -m unittest discover
+    coverage run --source oauthlib -m pytest
```

Comment:
I think we should use the pytest-cov plugin instead.

Reply:
That's also possible right.
My concern was to have fewer changes as possible but if we want to use it I can make the change.

I also thought about adding a junit XML report to better see tests results history in CI...